src/vivarium_public_health/risks/base_risk.py

```diff
@@ -22,12 +22,28 @@ class Risk:
     (2) configuration:
             risk:
                 exposure: proxy_covariate
+
+    For polytomous risks, you can also provide an optional rebin block in the
+    configuration to indicate that the risk should be rebinned into a dichotomous
+    risk. That rebin block should contain 'exposed', a list of the categories
+    that should be rebinned into a single exposed category in the resulting
+    dichotomous risk. For example, for a risk named "risk" with categories
+    cat1, cat2, cat3, and cat4 that you wished to rebin into a dichotomous risk
+    with an exposed category containing cat1 and cat2 and an unexposed category
+    containing cat3 and cat4, the configuration could look like this:
+
+    configuration:
+        risk:
+            rebin:
+                exposed: ['cat1', 'cat2']
     """
 
     configuration_defaults = {
         "risk": {
             "exposure": 'data',
-            "rebin": {},
+            "rebin": {
+                "exposed": [],
+            },
             "category_thresholds": [],
```

Comment:
what is category_thresholds for? I was going to add it to the docstring when I updated it for rebin but I couldn't remember what this was

Reply:
They are thresholds we use to bin the continuous distributions for alternative risks.

Reply:
They should be mutually exclusive with the rebin arguments.

Reply:
ok I added a bit to the docstring about this. will you make sure I represented it correctly? And I added a check to the rebinned validator to make sure that category thresholds aren't also supplied